### DISCUSSION TITLE
Suggestion: More descriptive shorthand errors

### DISCUSSION THREAD

**Discussion by benschac** [tamagui/tamagui#2828] [General]
### Current Behavior

https://github.com/tamagui/takeout/issues/302#issue-2204515857

prop is just, not found

### Expected Behavior

in the typescript error getting something like:
shorthand mode is enabled. `fontWeight` is aliased to `fow`

### Tamagui Version

```markdown
all
```

### Platform (Web, iOS, Android)

```markdown
all
```

### Reproduction

```markdown
n/a
```

### System Info

```markdown
n/a
```

**Comment by natew** [comment]
Typescript is impossible to really solve, but runtime could, moving to discussion.
